templates/helm/orchestration-workflows/hca/templates/workflow.yaml

```diff
@@ -56,18 +56,20 @@ spec:
       useFlexRS: {{ eq .Values.env "prod" }}
     bigquery:
       stagingData:
-        project: {{ $projectName }}
+        project: {{ $projectId }}
```

Comment:
❓ Can you describe what this change is for?

Reply:
SO just about every project we've made in GCP, we've actually made ourselves, which means the project name and the project id line up (like `broad-dsp-monster-hca-dev` is both the id and name). This project was NOT made by us, the the person who made it let the ID be generated. Since the name and id are different, I had to make sure to use each where appropriate. This is one of the spots where it needs to be project id instead of name 🤷